test/thrift/parser_test.exs

```diff
@@ -1,8 +1,10 @@
 defmodule ParserTest do
-
   use ExUnit.Case, async: true
```

Comment:
Woops.  I copy/pasta'd the directory name.  Fixed that and changed the `setup` block to be `setup_all`.  That should work as long as someone doesn't reuse the same file name in multiple tests, right?